released/packages/coq-infotheo/coq-infotheo.0.4.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/infotheo"
+dev-repo: "git+https://github.com/affeldt-aist/infotheo.git"
+bug-reports: "https://github.com/affeldt-aist/infotheo/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Discrete probabilities and information theory for Coq"
+description: """
+Infotheo is a Coq library for reasoning about discrete probabilities,
+information theory, and linear error-correcting codes."""
+
+build: [
+  [make "-j%{jobs}%" ]
+  [make "-C" "extraction" "tests"] {with-test}
+]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.15" & < "8.17~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.14.0" & < "1.16~") | (= "dev") }
+  "coq-mathcomp-fingroup" { (>= "1.14.0" & < "1.16~") | (= "dev")  }
+  "coq-mathcomp-algebra" { (>= "1.14.0" & < "1.16~") | (= "dev")  }
+  "coq-mathcomp-solvable" { (>= "1.14.0" & < "1.16~") | (= "dev")  }
+  "coq-mathcomp-field" { (>= "1.14.0" & < "1.16~") | (= "dev")  }
+  "coq-mathcomp-analysis" { (>= "0.5.4") & (< "0.6~")}
+  "coq-hierarchy-builder" { >= "1.3.0" }
+]
+
+tags: [
+  "keyword:information theory"
+  "keyword:probability"
+  "keyword:error-correcting codes"
+  "keyword:convexity"
+  "logpath:infotheo"
+  "date:2022-11-01"
+]
+authors: [
+  "Reynald Affeldt, AIST"
+  "Manabu Hagiwara, Chiba U. (previously AIST)"
+  "Jonas Senizergues, ENS Cachan (internship at AIST)"
+  "Jacques Garrigue, Nagoya U."
+  "Kazuhiko Sakaguchi, Tsukuba U."
+  "Taku Asai, Nagoya U. (M2)"
+  "Takafumi Saikawa, Nagoya U."
+  "Naruomi Obata, Titech (M2)"
+]
+url {
+  http: "https://github.com/affeldt-aist/infotheo/archive/0.4.0.tar.gz"
```

Comment:
```suggestion
  src: "https://github.com/affeldt-aist/infotheo/archive/0.4.0.tar.gz"
```
According to the opam manual, `http` here is derecated (even if the linter doesn't catch it).